lib/pfconfig/namespaces/resource/UriFilters.pm

```diff
@@ -1,44 +1,33 @@
-package pfconfig::namespaces::resource::Profile_Filters;
-
+package pfconfig::namespaces::resource::UriFilters;
 =head1 NAME
 
-pfconfig::namespaces::resource::Profile_Filters
+pfconfig::namespaces::resource::UriFilters
```

Comment:
Julien we have been inconsistent with how we name the resources.
Sometimes they are lowercase other time they are mixed case.
Which standard would you prefer to use and I will change the new resources in this pull request to use that standard.